EazyE2E/Element/EzText.cs

```diff
@@ -0,0 +1,350 @@
+
+            if (Equals(attribute, TextPattern.MarginBottomAttribute))
+                _marginBottom = new EzTextResult<double> { IsMixed = instance.IsMixed, NotSupported = instance.NotSupported, Result = (double)Convert.ChangeType(instance.Result, typeof(T)) };
+
+            if (Equals(attribute, TextPattern.OutlineStylesAttribute))
+                _outlineStyle = new EzTextResult<OutlineStyles> { IsMixed = instance.IsMixed, NotSupported = instance.NotSupported, Result = (OutlineStyles)Convert.ChangeType(instance.Result, typeof(T)) };
+
+            if (Equals(attribute, TextPattern.OverlineColorAttribute))
+                _overlineColor = new EzTextResult<int> { IsMixed = instance.IsMixed, NotSupported = instance.NotSupported, Result = (int)Convert.ChangeType(instance.Result, typeof(T)) };
+
+            if (Equals(attribute, TextPattern.OverlineStyleAttribute))
+                _overlineStyle = new EzTextResult<TextDecorationLineStyle> { IsMixed = instance.IsMixed, NotSupported = instance.NotSupported, Result = (TextDecorationLineStyle)Convert.ChangeType(instance.Result, typeof(T)) };
+
+            if (Equals(attribute, TextPattern.StrikethroughColorAttribute))
+                _strikethroughColor = new EzTextResult<int> { IsMixed = instance.IsMixed, NotSupported = instance.NotSupported, Result = (int)Convert.ChangeType(instance.Result, typeof(T)) };
+
+            if (Equals(attribute, TextPattern.StrikethroughStyleAttribute))
+                _strikethroughStyle = new EzTextResult<TextDecorationLineStyle> { IsMixed = instance.IsMixed, NotSupported = instance.NotSupported, Result = (TextDecorationLineStyle)Convert.ChangeType(instance.Result, typeof(T)) };
+
+            if (Equals(attribute, TextPattern.UnderlineColorAttribute))
+                _underlineColor = new EzTextResult<int> { IsMixed = instance.IsMixed, NotSupported = instance.NotSupported, Result = (int)Convert.ChangeType(instance.Result, typeof(T)) };
+
+            if (Equals(attribute, TextPattern.UnderlineStyleAttribute))
+                _underlineStyle = new EzTextResult<TextDecorationLineStyle> { IsMixed = instance.IsMixed, NotSupported = instance.NotSupported, Result = (TextDecorationLineStyle)Convert.ChangeType(instance.Result, typeof(T)) };
+
+            if (Equals(attribute, TextPattern.TabsAttribute))
+                _tabStops = new EzTextResult<double[]> { IsMixed = instance.IsMixed, NotSupported = instance.NotSupported, Result = (double[])Convert.ChangeType(instance.Result, typeof(T)) };
+
+            if (Equals(attribute, TextPattern.TextFlowDirectionsAttribute))
+                _textFlowDirection = new EzTextResult<FlowDirections> { IsMixed = instance.IsMixed, NotSupported = instance.NotSupported, Result = (FlowDirections)Convert.ChangeType(instance.Result, typeof(T)) };
+        }
```

Comment:
@abduncan Is there an easier way to do this?  I tested it and it works so it's not particularly urgent but it we ever want to add more properties to this class, we'll have to remember to add it here.